### ISSUE TITLE
Enhancement/QOLI: Invert Hide Selected Function - Should hide unselected (Interaction prototype included)

### ISSUE THREAD

**Issue by FoxInTheB0xFFF** [mitre-attack/attack-navigator#571]
Hi there,

tldr; The interaction and/or solution isn't immediately obvious and the base interaction doesn't behave as expected.

There was a post a while back that I found after having a frustrating experience with the attack-navigator… Figured I would rehash the conversation as this really didn't make sense to me. Here is the original issue and the remarks w/Prototype

https://github.com/mitre-attack/attack-navigator/issues/57

I was running the matrix locally for a demo and ran into the same issues. Although it does make sense to hide-selected, the interaction is backwards.

Current Hide Workflow: ```Select tiles -> Select 'Disable' -> Select 'HIDE Disabled'```
* This workflow requires selections above, below, and across a much larger set of touch-points
* It doesn't follow common selection pattern (e.g. We select the apples we would like to purchase)
* The Enabled/Disabled states are "modal" - modes tend to introduce errors
* This pattern doesn't follow similar patterns in the app (e.g. Select a tile to apply a score)
* Additionally the order of the layer controls could be improved
    * Grouping controls often used in conjunction with each other closer together - Law of proximity
    * Using icons that are more familiar to users - Recognition requires less cognitive load than recollection
Expected: Select tiles -> Select 'HIDE Unselected
https://www.figma.com/proto/rJXqzMT0YIfMa0vn4qIztQ/Untitled?page-id=0%3A1&type=design&node-id=1-5&viewport=1884%2C781%2C1.32&t=sj88relpeZJUomwb-1&scaling=min-zoom&mode=design

Hope this helps, as this was quite frustrating for me, especially after I accidentally selected a single item and lost my previous selections. This resulted in having to start all over.

**Comment by clemiller** [comment]
Hi @FoxInTheB0xFFF,

Thank you for the suggestions! We recognize the process for hiding techniques from the matrix view could be improved and your comments are very helpful in understanding the areas that cause frustration. We will look into tackling this in the upcoming year.

On a related note, we have an issue open to improve the Navigator toolbar (#534), which aims to make the various controls more user friendly. I am linking it here so we can take your feedback for the layer controls into consideration when we start working on those improvements.

I'll leave this issue open and mark it as "seeking feedback" in case others would like to share their thoughts as well!
